x-pack/plugins/secops/public/components/timeline/body/column_headers/range_picker/index.tsx

```diff
@@ -17,7 +17,7 @@ interface Props {
   onRangeSelected: OnRangeSelected;
 }
 
-export const RangePickerWidth = 110;
+export const RangePickerWidth = 120;
```

Comment:
So much React and styled going on it's easy to forget that typically variables don't start with an upper case letter. I would consider using normal variable naming convention with lowercase or all CAPS.